tests/e2e/core-tests/README.md

```diff
@@ -0,0 +1,100 @@
+# WooCommerce Core End to End Test Suite
+
+This package contains the automated end-to-end tests for WooCommerce.
+
+## Table of contents
+
+- [Pre-requisites](#pre-requisites)
+- [Setting up core tests](#setting-up-core-tests)
+- [Test functions](#test-functions)
+  - [Activation and setup](#activation-and-setup)
+  - [Merchant](#merchant)
+  - [Shopper](#shopper)
+- [Contributing a new test](#contributing-a-new-test)
+
+## Pre-requisites
+
+### Setting up the test environment
+
+Follow [E2E setup instructions](https://github.com/woocommerce/woocommerce/tests/e2e/README.md).
```

Comment:
Sorry, link still doesn't seem to work, think it needs to be `(https://github.com/woocommerce/woocommerce/blob/master/tests/README.md)` or `(../tests/e2e/README.md)` if used a relative link

Reply:
@tammullen Relative links don't work on https://npmjs.com. Updated to include `blob/master/`.